tests/MongoDB.Bson.TestHelpers/JsonDrivenTests/JsonDrivenTestCaseFactory.cs

```diff
@@ -51,6 +51,7 @@ IEnumerator IEnumerable.GetEnumerator()
 
         protected virtual BsonDocument ReadJsonDocument(string path)
         {
+            BsonDefaults.GuidRepresentation = GuidRepresentation.Standard;
```

Comment:
without this line some json tests cannot be parsed

Reply:
OK.

But we're changing a global default half way through running the tests. That would seem like a race condition with unpredictable side effects.

Reply:
Removed